src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -86,8 +86,12 @@ async fn main() {
                 "Ingest a file by path, which allows it to be used in Directories/Snapshots. \
 Outputs a fingerprint of its contents and its size in bytes, separated by a space.",
               )
-              .arg(Arg::new("path").required(true).takes_value(true))
-              .arg(Arg::new("output-mode").long("output-mode").possible_values(["json", "simple"]).default_value("simple").multiple_occurrences(false).takes_value(true).help(
```

Comment:
multiple_occurrences is the default (or rather, it corresponds to `ArgAction::Set`, which is the default).